packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -218,7 +237,14 @@ contract L2ToL2CrossDomainMessenger is ISemver, TransientReentrancyAware {
     function _decodeSentMessagePayload(bytes calldata _payload)
         internal
         pure
-        returns (uint256 destination_, address target_, uint256 nonce_, address sender_, bytes memory message_)
+        returns (
+            uint256 destination_,
+            address target_,
+            uint256 nonce_,
+            address sender_,
+            bytes memory message_,
+            address entrypoint_
+        )
```

Comment:
Since we are modifying it, should we add the NATSPEC as well?

Reply:
Ok, some internals have natspec, some have natspec without params and some use inherit.